### PR TITLE
Update tmuxp

### PR DESCRIPTION
**Pull request by egorenar**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64 + glibc

### CODE REVIEW

File: srcpkgs/python3-libtmux/template [hunk 1]
```diff
@@ -10,7 +10,7 @@ maintainer="Alexander Egorenkov <egorenar-dev@posteo.net>"
 license="MIT"
 homepage="https://github.com/tmux-python/libtmux"
 distfiles="${PYPI_SITE}/l/libtmux/libtmux-${version}.tar.gz"
-checksum=d82cf391097eb69d784d889d482bb99284b984aa6225276a3dc1af8c1460dd3c
+checksum=c40959c5ae9e7add705a3c4e60365667e65ab45835a9e663692e428e1d7e42e2
 # Disable check which uses pip to pull external python packages
 make_check=no
```
Comment (paper42) [inline srcpkgs/python3-libtmux/template:15]: The reason is wrong and at the time it was added it was probably just a missing dependency.
```suggestion
checkdepends="python3-pytest tmux"
# fixture 'config_file' not found
make_check=no
```

**Comment by icp1994** [comment]
https://github.com/void-linux/void-packages/pull/40193

File: srcpkgs/python3-libtmux/template (hunk 1 above)
Comment (egorenar) [reply srcpkgs/python3-libtmux/template]: thanks! fixed.

**Comment by paper42** [comment]
https://github.com/void-linux/void-packages/pull/40193

**Pull request closed without merging**